src/index.ts

```diff
@@ -96,15 +114,23 @@ export class DynamoDBAutoIncrement {
           Item: {
             ...this.props.counterTableKey,
             [this.props.counterTableAttributeName]: nextCounter,
+            ...lastItem,
```

Comment:
No, you want to update this with `item`, not `lastItem`.

Reply:
No, in this case I want to save the last item. The previous version for the item being updated will get saved in the PUT to `this.props.counterTableName` (lines 109-120) , and the new version with the updated counter value gets saved in the PUT to `this.props.tableName` (121-135)

Reply:
Don't we want the _current_ version and _current_ item contents saved to both tables?

Reply:
No, I don't think it makes sense to save the data twice, right? Queries against the history table should only be retrieving old versions of the data. This was at least the original assumption made in the demo version.

Reply:
I prefer to store the current version in both tables. It simplifies the access pattern for retrieving a document by version number because it requires only one request instead of possibly two. It also makes this a one-line change. That's what I had in mind with #19.

Reply:
Okay that should be an easier implementation, I'll updated the PR